frontend/src/utilities/notebookControllerUtils.ts

```diff
@@ -356,14 +356,8 @@ export const getNotebookEventStatus = (
         };
     }
   }
-
+  //for events not related to oauth
```

Comment:
```suggestion
  // For notebook events
```
Best to call out what they are -- it's a 2 container system -- although you are technically correct, it's a bit more helpful to say they are notebook related to run home the goal of this -- honestly an "else" between the two blocks is probably better lol... but no need to do that here.